backend/gateway/mux/mux.go

```diff
@@ -62,14 +69,53 @@ func (a *assetHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 	// Serve!
 	if f, err := a.fileSystem.Open(r.URL.Path); err != nil {
+		// If not a known static asset and an asset provider is configured, try streaming from the configured provider.
+		if a.assetCfg != nil {
+			if a.assetCfg.Provider != nil && strings.HasPrefix(r.URL.Path, "/static") {
+				// We attatch this header simply for observibility purposes.
+				// Otherwise its difficult to know if the assets are being served from the configured provider.
+				w.Header().Set("x-clutch-asset-passthrough", "true")
+
+				// TODO: handle errors
```

Comment:
why is this a TODO? only a few lines of code to handle err unless i'm missing something

Reply:
Here is the original context. https://github.com/lyft/clutch/pull/468#discussion_r496060061

Reply:
How would you suggest handling these errors? Panic? throw 5xx back to the user? Or have a better 5xx error page?

Reply:
throw 5xx, it's at least debuggable in browser tools. we can fix up with nicer error page later if it makes sense.